recipes/openvr/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class OpenvrConan(ConanFile):
+    name = "openvr"
+    description = "API and runtime that allows access to VR hardware from applications have specific knowledge of the hardware they are targeting."
+    topics = ("conan", "openvr", "vr", )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ValveSoftware/openvr"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows" or self.options.shared:
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, "11")
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
```

Comment:
Not even GCC 4.9 is able to build?

Reply:
Openvr use the <codecvt> header which is only included in gcc 5 and later.